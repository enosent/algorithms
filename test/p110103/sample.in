3
10.00
20.00
30.00
4
15.00
15.01
3.00
3.01
3
10.00
10.00
10.00
4
10.00
10.00
10.00
10.00
5
10.01
10.01
10.00
10.00
10.00
5
10.01
10.01
10.01
10.00
10.00
5
10.01
10.01
10.01
10.01
10.00
6
10.01
10.01
10.00
10.00
10.00
10.00
6
10.01
10.01
10.01
10.00
10.00
10.00
6
10.01
10.01
10.01
10.01
10.00
10.00
6
10.01
10.01
10.01
10.01
10.00
10.00
5
10.02
10.00
10.00
10.00
10.00
5
10.03
10.00
10.00
10.00
10.00
0