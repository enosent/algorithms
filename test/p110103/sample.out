$10.00
$11.99
$0.00
$0.00
$0.00
$0.00
$0.00
$0.00
$0.00
$0.00
$0.00
$0.01
$0.02